data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -69,6 +69,16 @@ frontend {{ front }}
 {%         endif %}
 {%         if front_config.mode is vyos_defined %}
     mode {{ front_config.mode }}
+{# add tcp-request related directive if ssl is configed #}
+{%             if front_config.mode is vyos_defined('tcp') and front_config.rule is vyos_defined %}
+{%                 for rule, rule_config in front_config.rule.items() %}
+{%                     if rule_config.ssl is vyos_defined %}
+    tcp-request inspect-delay 5s
+    tcp-request content accept if {{ "{" }} req_ssl_hello_type 1 {{ "}" }}
```

Comment:
Why you need to place the `{` in a Jinja2 statement without variables?

here is no LINTing error using: `tcp-request content accept if { req_ssl_hello_type 1 }`

Reply:
changed to `tcp-request content accept if { req_ssl_hello_type 1 }` in the latest commit